ams/CleanupCommand.cs

```diff
@@ -0,0 +1,233 @@
+﻿using AMSMigrate.Ams;
+using AMSMigrate.Contracts;
+using AMSMigrate.Transform;
+using Azure;
+using Azure.Core;
+using Azure.ResourceManager.Media;
+using Azure.Storage.Blobs;
+using Microsoft.Extensions.Logging;
+using Microsoft.Identity.Client.Extensions.Msal;
+using Spectre.Console;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace AMSMigrate.ams
+{
+    internal class CleanupCommand : BaseMigrator
+    {
+        private readonly ILogger _logger;
+        private readonly CleanupOptions _options;
+        private readonly IMigrationTracker<BlobContainerClient, AssetMigrationResult> _tracker;
+
+        public CleanupCommand(GlobalOptions globalOptions,
+            CleanupOptions cleanupOptions,
+            IAnsiConsole console,
+            TokenCredential credential,
+            IMigrationTracker<BlobContainerClient, AssetMigrationResult> tracker,
+            ILogger<CleanupCommand> logger)
+            : base(globalOptions, console, credential)
+        {
+            _options = cleanupOptions;
+            _logger = logger;
+            _tracker = tracker;
+        }
+        public override Task MigrateAsync(CancellationToken cancellationToken)
+        {
+            return Task.FromResult<object>(null);
+        }
+
+        public async Task CleanUpAsync(CancellationToken cancellationToken)
```

Comment:
Instead of keeping an empty MigrateAsync( ) with build warnings,  I wonder if you can move the CleanupAsync into MigrateAsync( ) and get rid of CleanupAsync( ) method here.  Since CleanupCommand derives from BaseMigrator class anyway, each derived migrator class has implementation of MigrateAsync() which does the actual work for this migrator.  For Cleanup command, its migrator work is to do the cleanup.